action/protocol/poll/protocol.go

```diff
@@ -269,6 +281,20 @@ func (p *governanceChainCommitteeProtocol) delegatesByGravityChainHeight(height
 			Votes:         c.Score(),
 			RewardAddress: rewardAddress,
 		})
+		n = append(n, c.Name())
```

Comment:
You may consider add Name to Candidate, but do not add it to proto definition if we don't need to materialize it

Reply:
yeah, i added to Candidate earlier but found need to modify protobuf. we can do this if omitting this field from proto